src/code/templates/genome-playground.js

```diff
@@ -18,13 +18,15 @@ export default class GenomeContainer extends Component {
     let penView = <div className='columns bottom'>
                     <PenView orgs={ keptDrakes } width={500} columns={5} rows={1} tightenRows={20}/>
                   </div>;
-    
+
+    // we don't increment the score, so users always get top mark from playground
+    const kNoScoreIncrement = false;
```

Comment:
Got it, perhaps I'm not familiar with the notation style, is it related to this? http://stackoverflow.com/questions/28762564/why-do-variable-names-start-with-the-letter-k

The way I read it makes it seem like `noScoreIncrement = false`, which means `scoreIncrement = true` by a double negative... is a constant `doesPlaygroundIncrementMoves` any clearer? I agree the code is confusing without any named constant- is it in general best practice to name parameters if their use is ambiguous?

These are very small nits, thank you for entertaining them! More than anything else I'm trying to figure out the best style to use myself moving forward.